feat(react-scheduler): support scheduler localization capability (#1985)
DevExpress/devextreme-reactive: packages/dx-react-scheduler/docs/reference/scheduler.md

-----|------|---------|------------
data | Array&lt;[AppointmentModel](#appointmentmodel)&gt; | | An array of appointment data objects.
rootComponent | ComponentType&lt;[Scheduler.RootProps](#schedulerrootprops)&gt; | | A component that renders the root layout.

> locale | string | 'en-US' | A string that specifies a locale for date formatting.

Reviewer: ```suggestion
locale | string | 'en-US' | The locale according to which dates should be formatted.
```
